at_lookup/test/secondary_address_cache_test.dart

```diff
@@ -1,15 +1,15 @@
 import 'package:at_commons/at_commons.dart';
-import 'package:at_lookup/src/cache/secondary_address_cache.dart';
+import 'package:at_lookup/src/cache/cacheable_secondary_address_finder.dart';
 import 'package:test/expect.dart';
 import 'package:test/scaffolding.dart';
 import 'package:mocktail/mocktail.dart';
 
-class MockSecondaryFinder extends Mock implements SecondaryAddressFinder {}
+class MockSecondaryFinder extends Mock implements SecondaryUrlFinder {}
 
 void main() async {
   String rootDomain = 'root.atsign.unit.tests';
   int rootPort = 64;
-  SecondaryAddressFinder mockSecondaryFinder = MockSecondaryFinder();
+  SecondaryUrlFinder mockSecondaryFinder = MockSecondaryFinder();
```

Comment:
Very very nice to see mock in action. Abstracting out external dependency into SecondaryUrlFinder makes this possible. Good work Murali. Yayyyy.

Reply:
the tests were written  by Gary as a part of initial checkin